package.json

```
@@ -33,7 +33,7 @@
     "LICENSE"
   ],
   "peerDependencies": {
-    "webpack": "^4.44.2"
+    "webpack": ">=4.44.2"
```

```suggestion
    "webpack": "^4.44.2 || ^5.46.0"
```

Using `>=` would allow v6 if/when that happens, which is not correct

---

Sure thing. My initial solution was to revert to the previous setting, but `^4.44.2 || ^5.46.0` is better.